app/models/application_record.rb

```diff
@@ -75,6 +75,23 @@ def pending?
     !reviewed
   end
 
+  def save_for_later(date = nil, note = nil, mail = false)
+    update_attributes(saved_for_later: true,
+                      date_for_later: date,
+                      note_for_later: note)
+    JobappsMailer.send_note_for_later(self).deliver_now if mail
+  end
+
+  def unsave
+    update_attributes(saved_for_later: false,
```

Comment:
No. The note can be replaced if necessary. Also, say you had saved an application in May for later review in September, and it got moved back to the dashboard. Would it not be helpful to see your note about why you'd saved it?